migrate/repo_checks.py

```diff
@@ -481,10 +481,11 @@ def fix(self, dry_run=False):
             if not dry_run:
                 try:
                     self.api.issues.create_label(
-                        self.org_name,
-                        self.repo_name,
-                        label,
-                        self.labels[label],
```

Comment:
The actual bug:

Before, `label` was just a dict name, and `self.labels` was a dict mapping name -> color.

After #60 , `label` is a dict, but I didn't update this API call to account for that.